app/views/works/form.html.erb

```diff
@@ -37,33 +37,12 @@
     <% end %>
 
     <% component.with_pane(tab_name: :dates, label: t('works.edit.panes.dates.label'), form:) do %>
-      <% form.fields_for :publication_date do |date_form| %>
```

Comment:
Pull out of the form view. Now uses the non-repeatable nested component HTML with the publication date edit component HTML, with the former being a reusable component.